### DISCUSSION TITLE
Remap built-it MacBook keyboard to use Miryoku

### DISCUSSION THREAD

**Discussion by alloc33** [manna-harbour/miryoku#125] [Q&A] [answered]
Is there any way to use Miryoku with built-in MacBook keyboard? 

![miryoku-kle-kmonad](https://user-images.githubusercontent.com/45129790/183853684-15976789-d13a-4368-b86d-8b8d3bce970f.png)

**Comment by alloc33** [comment]
I think I found solution - KMonad Xd

**Comment by manna-harbour** [comment, marked as answer]
Yes, you can use Miryoku KMonad with non-programmable keyboards: https://github.com/manna-harbour/miryoku_kmonad. It's best with ergo keyboards, but the lite mapping can be used with laptop keyboards: https://github.com/manna-harbour/miryoku_kmonad#lite. 

You can see some new diagrams in https://github.com/manna-harbour/miryoku/pull/106, e.g. https://raw.githubusercontent.com/manna-harbour/miryoku/2ef12f3bff33bbe25ac47933084a482e69c91d86/data/layers/miryoku-kle-reference-lite-qwerty-ansi.png and https://raw.githubusercontent.com/UncleCJ/miryoku/b593099872ee0a6c617228731f561ca8fb5379c9/data/layers/miryoku-kle-reference-lite-qwerty-vi-iso-apple.png.
